web/source/osk/languageMenu.ts

```diff
@@ -411,23 +411,17 @@ namespace com.keyman.osk {
       // We're setting up a few events - this alias helps avoid scoping issues.
       const languageMenu = this;
 
-      // Touchstart (or mspointerdown) event highlights the touched list item
-      const touchStart=function(e) {
-        e.stopPropagation();
-        if(this.className.indexOf('selected') <= 0) {
-          this.className=this.className+' selected';
-        }
-        languageMenu.scrolling=false;
-        languageMenu.y0=e.touches[0].pageY;//osk.lgList.childNodes[0].scrollTop;
-
+      // Refer to https://github.com/keymanapp/keyman/pull/7790 for context on
+      // the following two methods.
+      const lockBodyScroll = () => {
```

Comment:
I think I'd like to make these two functions more robust by ensuring that repeated calls don't lead to broken behaviour -- as it stands, if you called `lockBodyScroll` twice before `unlockBodyScroll`, it would then never be able to unlock :grin:

I don't think this can happen at present, but it's good for code durability in the long run.